documentation/modules/deploying/con-deploy-product-downloads.adoc

```diff
@@ -16,7 +16,8 @@ endif::Downloading[]
 Strimzi release artifacts include sample YAML files to help you deploy the components of Strimzi to Kubernetes, perform common operations,
 and configure your Kafka cluster.
 
-You deploy Strimzi to a Kubernetes cluster using the `kubectl` command-line tool.
+Use `kubectl` to deploy Strimzi Cluster Operator from the `install/cluster-operator` folder from the downloaded zip file.
+In addition, if you want to use stand-alone installations of the Topic and User Operators with a Kafka cluster not managed by the Strimzi Cluster Operator, you can deploy them from the `install/topic-operator` and `install/user-operator` folders.
```

Comment:
```suggestion
In addition, if you want to use standalone installations of the Topic and User Operators with a Kafka cluster that is not managed by the Strimzi Cluster Operator, you can deploy them using the YAML files in the `install/topic-operator` and `install/user-operator` folders.
```

Reply:
NOw we have expanded this download section, I wonder if we now need a link down to https://strimzi.io/docs/operators/master/deploying.html#deploy-create-cluster_str in case the reader assumes these are all the instructions they need?